### DISCUSSION TITLE
Tinycam, how come v3 works without internet?

### DISCUSSION THREAD

**Discussion by gitlot6** [gtxaspec/wz_mini_hacks#811] [Q&A]
Perhaps there is simpler way? Anybody looked into tinycam, how he sends stream?
Tinycam needs internet to start stream from v3, but after that you can disconnect internet and stream continues

**Comment by DAMAGEDatheCORE** [comment]
That's just how it works, even with stock firmware. It needs to initially
connect with WYZE's servers to authenticate, but after that the cam cam be
disconnected from the internet. If it loses power or you restart it, you
have to connect again to reauthenticate.
